backend/server.js

```diff
@@ -1,61 +1,63 @@
-const cors = require("cors");
-const morgan = require("morgan");
-const express = require("express");
-const SequelizeStoreInit = require("connect-session-sequelize");
-const session = require("express-session");
-const Sequelize = require("sequelize");
-const app = express();
-const PORT = 3000;
+const cors = require('cors');
+const morgan = require('morgan');
+const express = require('express');
+const SequelizeStoreInit = require('connect-session-sequelize');
+const session = require('express-session');
+const Sequelize = require('sequelize');
+const app = express()
 
-app.use(
-  cors({
-    origin: "http://localhost:5173",
-    credentials: true,
-  }),
-);
-app.use(express.json());
-app.use(morgan());
+app.use(cors({
+  origin: 'http://localhost:5173',
+  credentials: true
+}));
+app.use(express.json())
+app.use(morgan())
 
-const newSequelize = new Sequelize("event_finder", "postgres", "1025323", {
-  host: "localhost",
-  dialect: "postgres",
+const newSequelize = new Sequelize('event_finder', 'postgres', '1025323', {
+    host: 'localhost',
+    dialect: 'postgres'
 });
 
 const SequelizeStore = SequelizeStoreInit(session.Store);
 const sessionStore = new SequelizeStore({
-  db: newSequelize,
+  db: newSequelize
 });
 
 // Session middleware
 app.use(
   session({
-    secret: "your-secret-key",
+    secret: 'your-secret-key',
     resave: false,
     saveUninitialized: false,
     store: sessionStore,
     cookie: {
       sameSite: false,
       secure: false,
-      expires: new Date(Date.now() + 365 * 24 * 60 * 60 * 1000), // 1 year in milliseconds
-    },
-  }),
+      expires: new Date(Date.now() + (365 * 24 * 60 * 60 * 1000)) // 1 year in milliseconds
+    }
+  })
 );
 sessionStore.sync();
```

Comment:
Is your sequelize db set up? It does not even have secret key right? CodePath example uses sequelize db to store user data rather than prisma db. And your backend is backed by prisma db. What is the purpose of your sequelize db here?

Reply:
To Do:
Delete sequelize db & pass content to prisma